OneSignalSDK/onesignal/core/src/main/java/com/onesignal/core/internal/operations/impl/OperationRepo.kt

```diff
@@ -183,6 +184,10 @@ internal class OperationRepo(
         }
     }
 
+    override fun forceProcessDeltas() {
```

Comment:
I think this should be called `forceExecuteOperations`, since we have the method name ` executeOperations` in this code base.